### ISSUE TITLE
Saml -> Google Applications skips Duo

### ISSUE THREAD

**Issue by Sepiidae** [epierce/cas-server-extension-duo#3]
We discovered that we are unable to perform duo authentication to our Google Applications site. Have you see this before? If you would like a copy of our login flow we will send it your way.

**Comment by epierce** [comment]
Yeah, that'd be great.  I haven't seen a problem with that here, but there are only a few of us that have Duo turned on for all sites
